test/Directory.Build.props

```diff
@@ -24,6 +24,10 @@
     <IncludeSymbols>false</IncludeSymbols>
   </PropertyGroup>
 
+  <ItemGroup>
+    <PackageReference Include="AwesomeAssertions" />
```

Comment:
I don't have a problem with having this here instead of in individual projects in theory, but it seems like it's failing in UnitTests.proj because that's targeting net40 (?). It's probably easiest to update that to use net472, but you could also just remove it there.

Reply:
Fixed the unittest.proj. The issue is that the .proj is not actually a real project and so was getting a default TFM of net40 (we apparently have code for that for legacy projects). That broke because of this implicit reference so I removed it explicitly from unittest.proj. Open to arguments as to which is better but this let me clean up a bunch of test projects so less overall content in the files.

Reply:
Hard for me to argue with you taking one of my two suggestions 🙂